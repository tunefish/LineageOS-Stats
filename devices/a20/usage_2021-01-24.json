{
  "17.1": 2416,
  "none": 6
}