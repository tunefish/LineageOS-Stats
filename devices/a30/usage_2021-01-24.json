{
  "17.1": 28,
  "18.0": 597,
  "18.1": 127,
  "none": 4
}