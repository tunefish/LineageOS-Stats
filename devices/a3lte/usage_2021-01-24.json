{
  "15.1": 473,
  "16.0": 163,
  "17.1": 562
}