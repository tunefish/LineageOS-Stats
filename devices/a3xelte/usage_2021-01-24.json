{
  "15.1": 96,
  "16.0": 415,
  "17.1": 2934,
  "18.0": 3,
  "18.1": 7,
  "52.8": 1,
  "none": 2
}