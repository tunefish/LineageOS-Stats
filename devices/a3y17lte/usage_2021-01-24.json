{
  "14.1": 5374,
  "15.1": 280,
  "16.0": 850,
  "17.1": 776,
  "18.0": 1
}