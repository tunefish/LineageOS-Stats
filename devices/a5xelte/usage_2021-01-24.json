{
  "15.1": 41,
  "16.0": 477,
  "17.1": 3546,
  "18.0": 12,
  "18.1": 1,
  "none": 1
}