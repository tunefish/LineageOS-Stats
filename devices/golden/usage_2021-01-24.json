{
  "13.0": 911,
  "14.1": 2856,
  "15.1": 4,
  "17.1": 2,
  "none": 1
}